### PR TITLE
Swapped to stabilised asm!() and global_asm!() macros

### PR DESCRIPTION
**Pull request by tobygomersall**

### CODE REVIEW

**Comment by tobygomersall** [comment]
Hi, If these changes are acceptable I'm keen to get this merged. If not, I'm happy to make corrections.

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@ Cargo.lock
 target/
 bin/*.after
 bin/*.before
-bin/*.o
\ No newline at end of file
+bin/*.o
+*.swp
```
Comment (ilya-epifanov) [inline .gitignore]: I think this line belongs in you user-wide git ignore

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "picorv32"
-version = "0.1.3"
+version = "0.1.4"
```
Comment (ilya-epifanov) [inline Cargo.toml]: I'd suggest 0.2.0.
Just in case it breaks something, at least people who use "0.1" will not be affected.

**Review by ilya-epifanov — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (tobygomersall) [reply .gitignore]: Removed *.swp

File: Cargo.toml (hunk 2 above)
Comment (tobygomersall) [reply Cargo.toml]: Updated to 0.2.0.

**Review by ilya-epifanov — APPROVED** [review]

**Pull request merged**